### DISCUSSION TITLE
How I can Change the Admin login password by without login in admin console

### DISCUSSION THREAD

**Discussion by ajeetkumarver** [keycloak/keycloak#35355] [Q&A]
I have to change Admin login pass , I forgot y current password So I am not able to login in my admin console , I have email register to that admin and admin username , So is there any way to change the password fro the admin using these details please help....!!!

Thanks

**Comment by jdevries-transheroes** [comment]
In the `credential` table you can set a new (hashed) password

**Comment by ajeetkumarver** [comment]
If I am making any changes in keycloack dB then I have to restart keycloack
to see the effect?

**Reply by jdevries-transheroes** [reply]
Yes

**Comment by ajeetkumarver** [comment]
Hello, please confirm one more thing , if I change my admin password from
Keycloack DB then it will effect my existing running application?

**Comment by ajeetkumarver** [comment]
Hello ,
If I am sending keycloak reset password link from email , and If I am
changing my google login password for that email will it affect keycloak?

thanks & regards
